### PR TITLE
feature/remove-dns-records

### PR DESCRIPTION
**Pull request by bishopbm1**
Added new actions and workflows to remove a DNS Record and all its related records and PTR records since those are not cleaned up automatically. Added new actions and workflows to test if a hostname is available for use. Added new actions and workflows to test credentials against the server. Iterated pack version and added to changelog.

### CODE REVIEW

**Review by nmaludy — CHANGES_REQUESTED** [review]
Missing README update explaining the new actions

**Comment by bishopbm1** [comment]
Updated Readme with explanation and example usages.

File: README.md [hunk 1]
```diff
@@ -291,8 +291,83 @@ result:
 
 ```
 
+## <a name="UsageAdditionalActions"></a> Usage - Additional Actions
+
+In addition to the actions above there are a few additional actions that we will discuss here.
```
Comment (nmaludy) [inline README.md]: "In addition to the auto-generated actions described above, there are a handful of higher-level workflow actions that implement common functionality."

File: README.md [hunk 2]
```diff
@@ -291,8 +291,83 @@ result:
...
+  dns_domain: domain.tld
+  hostname: test-name-2
+  password: '***'
+  server: menandmice.domain.tld
+  username: administrator
+status: succeeded
+start_timestamp: Wed, 01 May 2019 19:28:20 UTC
+end_timestamp: Wed, 01 May 2019 19:28:30 UTC
+```
+
```
Comment (nmaludy) [inline README.md:385]: missing resolve_fqdn

**Comment by bishopbm1** [comment]
Ah yea sorry forgot about resolve_fqdn.. should be good now though.

**Review by nmaludy — APPROVED** [review]

**Pull request merged**